app/features-json/setting_json_controller.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "api_controller"
+
+module FastlaneCI
+  # Controller for providing all data relating to system settings
+  class SettingJSONController < APIController
+    HOME = "/data/settings"
+
+    get HOME do
+      return json(Services.setting_service.settings)
+    end
+
+    post "#{HOME}/:setting_key" do
+      key = params[:setting_key]
+      value = params[:value]
+
+      setting = Services.setting_service.find_setting(setting_key: key.to_sym)
+      return json({ status: :key_not_found }) if setting.nil?
+
+      # TODO: security aspect, how do we make sure this can't be abused?
+      # We don't want to hash/encrypt all of them, as this would make it harder for
+      # people to manually update those files
+      begin
+        setting.value = value
+        Services.setting_service.update_setting!(setting: setting)
+
+        return json({ status: :success })
+      rescue StandardError => ex
+        return json({ status: :error, message: ex.to_s })
```

Comment:
This will return with a 200 status. Sinatra providers an error handler, but we don't use it yet. I think we should let the error bubble up and then we add an error handler in `APIController` that will return a JSON error, with the proper status codes.